utils/getLinkData.js

```diff
@@ -0,0 +1,25 @@
+const { getLinkPreview } = require("link-preview-js");
+const { redisCache } = require("../config/cache");
+
+async function getLinkData(article_url) {
+  let results;
+  let isCached = false;
+
+  try {
+    const cacheResults = await redisCache.get(article_url);
+    if (cacheResults) {
+      isCached = true;
+      results = JSON.parse(cacheResults);
+      console.log("cache hit");
```

Comment:
Consider removing these *or* expanding them so that they are genuinely useful in the long run.  Can explain more in person.